tf_autolog_tests/tf1keras.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Hm should we delete this file / what's its purpose?

Reply:
Added to the commit because of `git add *keras.py` :) Resolved.